pkg/check/admin-check/admin_check.go

```diff
@@ -0,0 +1,196 @@
+package admin_check
+
+import (
+	"database/sql"
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	"github.com/juju/errors"
+	"github.com/ngaut/log"
+	"github.com/pingcap/tipocket/pkg/control"
+	"github.com/pingcap/tipocket/pkg/core"
+	"github.com/pingcap/tipocket/pkg/util"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+// AdminChecker returns a adminChecker instance
+func AdminChecker(cfg *control.Config) core.Checker {
+	return adminChecker{cfg: cfg}
+}
+
+const tiDBGCTime = 720 * time.Hour
+
+var ignoreDatabases = []string{"INFORMATION_SCHEMA", "PERFORMANCE_SCHEMA", "mysql"}
+
+type adminChecker struct {
+	cfg *control.Config
+}
+
+func (ac adminChecker) Check(_ core.Model, _ []core.Operation) (bool, error) {
+	if len(ac.cfg.ClientNodes) == 0 {
+		return false, errors.New("client node is empty")
+	}
+
+	index := rand.Intn(len(ac.cfg.ClientNodes))
+	node := ac.cfg.ClientNodes[index]
+	log.Info(fmt.Sprintf("root@tcp(%s:%d)/test", node.IP, node.Port))
+	db, err := sql.Open("mysql", fmt.Sprintf("root@tcp(%s:%d)/test", node.IP, node.Port))
+	if err != nil {
+		return false, nil
+	}
+
+	if err := adjustTiDBGC(db); err != nil {
```

Comment:
is calling adjustTiDBGC here later? `Check` is called after each each round of workload is finished?

Reply:
`adjustTiDBGC` is just to prevent errors when performing `admin check`